test/AutoDiff/Sema/DerivedConformances/derived_zero_tangent_vector_initializer.swift

```diff
@@ -0,0 +1,106 @@
+// RUN: %target-swift-frontend -emit-silgen %s | %FileCheck %s
+
+// Check `Differentiable.zeroTangentVectorInitializer` derivation.
+// There are two case:
+// 1. Memberwise derivation.
+// 2. `{ TangentVector.zero }` fallback derivation.
+
+import _Differentiation
+
+struct MemberwiseTangentVector: Differentiable {
```

Comment:
This PR seems to be missing `class` tests.

Reply:
Added `zeroTangentVectorInitializer` SILGen tests for classes and an enum.